discojs/src/dataset/preprocess.ts

```diff
@@ -0,0 +1,26 @@
+import * as tf from '@tensorflow/tfjs'
+import { TrainingInformation } from '../task/training_information'
+
+export type PreprocessImage = (image: tf.Tensor3D) => tf.Tensor3D
+
+export enum Preprocess {
+  Normalize = 'normalize',
```

Comment:
best to call them `image.normalize` or sth like this, as there might be a ton of functions in the future? btw there is still this `threshold` param on the task def which isn't saying what it does, i forgot what it's used for

Reply:
It's not used anywhere, I think this was part of sync Federated, where we had the threshold of how many fractions of users to choose from for training. I will delete it since it's deprecated.